janusgraph-core/src/main/java/org/janusgraph/graphdb/olap/QueryContainer.java

```diff
@@ -118,14 +112,7 @@ private QueryBuilder() {
             super(QueryContainer.this.tx);
         }
 
-        private Query relations(RelationCategory returnType) {
-//            if (name==null) {
-//                if (hasSingleType()) name = getSingleType().name();
-//                else if (!requiresName) name = QUERY_NAME_PREFIX + queries.size();
-//                else throw new IllegalStateException("Need to specify an explicit name for this query");
-//            }
-
-            BaseVertexCentricQuery vq = super.constructQuery(returnType);
+        private Query relations(RelationCategory returnType) {            BaseVertexCentricQuery vq = super.constructQuery(returnType);
```

Comment:
Split this line, please

Reply:
good catch

Reply:
+1 -1920 lines
The +1 made me curious :D